components/post_view/post_time.jsx

```diff
@@ -48,13 +48,17 @@ export default class PostTime extends React.PureComponent {
     renderTimeTag() {
         const date = new Date(this.props.eventTime);
 
+        const hour = this.props.useMilitaryTime ? date.getHours() : (date.getHours() % 12);
```

Comment:
`date.getHours() % 12` isn't right for [12AM, 1AM) or [12PM, 1PM). Should be `date.getHours() % 12 || 12`.

Reply:
thanks, updated